Storage.md

```diff
@@ -0,0 +1,209 @@
+This guide will describe how FullScreenShenanigans projects keep track of game information.
+
+### Table of Contents
+1. [ItemsHoldr](#itemsholdr)
+    1. [Local Storage](#local-storage)
+    2. [Items](#items)
+    3. [Updating](#updating)
+    4. [Clearing and Defaults](#clearing-and-defaults)
+    5. [Elements](#elements)
+2. [StateHoldr](#stateholdr)
+
+# ItemsHoldr
+
+ItemsHoldr is a versatile container to store and manipulate values in localStorage, and optionally keep an HTML container to show these values.
+
+## Local Storage
+
+ItemsHoldr keeps track of items and their values while the game is running.
+It can also save this information to `localStorage`.
+Other websites also use `localStorage`, so ItemsHoldr has a prefix property to differentiate which information belongs to which source.
```

Comment:
> which information belongs to which source

It would be more accurate to say that the prefix differentiates this ItemHoldr's information.